packages/react/src/components/Tabs/Tabs.stories.mdx

```diff
@@ -59,16 +75,34 @@ Import and use the `Tabs` component in your components as follows.
   format
   code={dedent`
 import Tab from '@oxygen-ui/react/Tab';
+import TabPanel from '@oxygen-ui/react/TabPanel';
 import Tabs from '@oxygen-ui/react/Tabs';\n
 function Demo() {
   return (
-    <Tabs value={value} onChange={handleChange}>
-      <Tab label="Tab One" />
-      <Tab label="Tab Two" />
-      <Tab label="Tab Three" />
-      <Tab label="Tab Four" />
-      <Tab label="Tab Five" />
-    </Tabs>
+    <Box>
+      <Tabs value={value} onChange={handleChange}>
+        <Tab label="Tab One" id="tab-0" aria-controls="tabpanel-0" />
```

Comment:
Any specific reason why there are 5 tabs?

Reply:
Actually there is no specific reason to add 5 tabs exactly. Anyway we can reduce the number of tabs.

Reply:
Shall we keep it at 3?

Reply:
Addressed with 3a5b291d9f4ff24274f8cdc7bb988596e2d4c935